<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSG</class>
 <widget class="QMainWindow" name="SSG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="menu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>메뉴</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="menuBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="map">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>지도</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mission">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>미션</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>저장</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>종료</string>
     </property>
    </widget>
    <widget class="QPushButton" name="time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>시간표</string>
     </property>
    </widget>
    <widget class="QPushButton" name="out_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>나가기</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="chat">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>741</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>벌써 점심시간이다. 점심먹으러 어디갈까?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>530</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>다음</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="hillBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="hillUpButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>언덕 위에 있는</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>언덕에 있는</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillDownButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>언덕 아래에 있는</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="hillUpMenuBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="hillUpMenu1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>안다미로 (한식)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillUpMenu2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>학생식당 (한식)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillUpMenu3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>블루포트 (카페)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillUpMenu4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>무아타 (분식)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="hillMenuOptionBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="hillMenuOption1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>한식</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenuOption2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>분식</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenuOption3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>빵류</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenuOption4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>기타</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="hillDownMenuBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="hillDownMenu1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>나성집 (한식)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillDownMenu2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>송스키친 (양식)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillDownMenu3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>원테이블 (양식)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillDownMenu4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>부암동 나들이</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="hillMenuBox1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="hillMenu1_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>The 함께식탁 (한식)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenu1_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>부대통령 뚝배기 (한식)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenu1_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>치즈밥있슈 (퓨전음식)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenu1_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>청와설렁탕 (곰탕, 설렁탕)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="hillMenuBox2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="hillMenu2_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>봉구스밥버거 (분식)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenu2_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>빨간계단 (종합분식)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenu2_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>윤가네 (종합분식)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenu2_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>신전떡볶이 (분식)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="hillMenuBox3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="hillMenu3_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>멜팅그릴 (햄버거)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenu3_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>샌드앤닭 (샌드위치)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenu3_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>파파존스 (피자)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="hillMenuBox4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="hillMenu4_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>GS25 (편의점)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenu4_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>아리가또맘마 (일식당)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenu4_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>네네치킨 (치킨, 닭강정)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hillMenu4_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>팔선생 (중식당)</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="nextButton2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>530</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>다음</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
